### PR TITLE
Add automatic style checks.

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```
Comment (ibauersachs) [inline .github/workflows/maven.yml:20]: Can we please do a matrix build with Java 8 and 11? See #15 for an example

File: pom.xml [hunk 2]
```diff
@@ -375,6 +375,21 @@
                     </instructions>
                 </configuration>
             </plugin>
+	    <plugin>
```
Comment (ibauersachs) [inline pom.xml:378]: Weird indents...

**Review by ibauersachs — CHANGES_REQUESTED** [review]

**Review by ibauersachs — DISMISSED** [review]
If you push your branch to this repo we'll see if they actually succeed. I'm not too familiar with the cache action, but I think it's necessary to include the Matrix parameters (i.e. the Java version) to the cache key.

**Comment by JonathanLennox** [comment]
> If you push your branch to this repo we'll see if they actually succeed. I'm not too familiar with the cache action, but I think it's necessary to include the Matrix parameters (i.e. the Java version) to the cache key.

I've copied the version from jitsi-utils instead.

**Review by ibauersachs — APPROVED** [review]

**Comment by JonathanLennox** [comment]
@ibauersachs Can you re-approve?

**Review by ibauersachs — APPROVED** [review]

**Pull request merged**